### DISCUSSION TITLE
Seeking Guidance as a New Wazuh User: Integrating Anomaly Detection Tool/DNN Model with Wazuh Pipeline

### DISCUSSION THREAD

**Discussion by Manos-Ar** [wazuh/wazuh#21624] [General]
Hello Wazuh community,

I am a newcomer to Wazuh. Currently, I am working on integrating my anomaly detection tool, which incorporates a Deep Neural Network (DNN) model, with the Wazuh pipeline. The primary function of my model is to parse network logs, conduct preprocessing, and perform inferences to identify anomalies. I am seeking guidance on how to seamlessly connect this solution with Wazuh to enhance our overall security monitoring capabilities. Given my limited experience with Wazuh, I would greatly appreciate any insights, best practices, or examples from the community. Thank you in advance for your assistance!

**Comment by n0obed** [comment]
Hi Manos,
Can I know if there is any update on your project?
I am working on a very similar project...
Looking for some head up from your experience.
